lib/parsers/lock-parser-base.ts

```diff
@@ -152,7 +152,7 @@ export abstract class LockParserBase implements LockfileParser {
         // TODO: also check the package version
         // for a stricter check
         if (strict) {
-          throw new OutOfSyncError(depName, this.type);
+          throw new OutOfSyncError(dep.name, this.type);
```

Comment:
the `depName` includes the version (i.e. `snyk@1.323.0`), while the `dep.name` only has the name (i.e. `snyk`)